appserver/tests/admin/tests/src/test/java/org/glassfish/main/admin/test/AsadminLoggingITest.java

```diff
@@ -69,11 +71,16 @@ public class AsadminLoggingITest {
 
     private static final Asadmin ASADMIN = GlassFishTestEnvironment.getAsadmin();
 
+    /** Fill up the server log. */
     @BeforeAll
-    public static void fillUpServerLog() {
-        // Fill up the server log.
-        AsadminResult result = ASADMIN.exec("restart-domain", "--timeout", "60");
-        assertThat(result, asadminOK());
+    public static void fillUpServerLog() throws Exception {
+        if (OS.isWindowsForSure()) {
+            // For some reason windows can collide on debug port.
```

Comment:
I don't experience blocked `--debug` ports on Linux in `restart-domain` command, but GHA on Windows does it. Seems like it doesn't close debug port when one process spawns another - the new then tries to open the same port, could reuse it, which is not a good idea, but ... 
I think we had this problem on Solaris many years ago too.